<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>WhitelistSettingsUI</class>
 <widget class="QDialog" name="WhitelistSettingsUI">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>300</width>
    <height>400</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Whitelist Settings</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <widget class="QListWidget" name="whitelistWidget"/>
   </item>
   <item>
    <widget class="QPushButton" name="addButton">
     <property name="text">
      <string>Add Character</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QPushButton" name="removeButton">
     <property name="text">
      <string>Remove Character</string>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>